Dockerfile

```diff
@@ -1,7 +1,7 @@
 ## relay-dapp dockerfile (This file is called Dockerfile) on server side. ##
-FROM node:12-alpine AS compiler
+FROM node:16-alpine AS compiler
 RUN apk add --no-cache build-base git bash
 WORKDIR /usr/src/app
 COPY . ./
-RUN npm i
-RUN npm start
+RUN npm i --no-audit
+# RUN npm start
```

Comment:
It's better to erase than comment, if the npm start is not needed please erase it.

Reply:
done